### ISSUE TITLE
Could be a sketch version problem? 

### ISSUE THREAD

**Issue by fuatkestane** [smartface/sketch-export-tool#2]
Hi,

Image does not match after importing sketch file. Looks different design.

**Sketch:** 

![image](https://user-images.githubusercontent.com/9901234/51317549-ec9ace00-1a68-11e9-9c36-a4d62f3147c4.png)

**Smartface UI:**

![image](https://user-images.githubusercontent.com/9901234/51317469-b52c2180-1a68-11e9-963b-8d805a305e7e.png)

My Sketch version: 52.4 (67378) 
 
Thanks.

**Comment by Burakdemirci** [comment]
- InfoArea label should be set multiline true on ide.
- continueButton the button does not support image on ide so you change the button to => flexlayout+label+imageview (flex layout for bacground and usabilty , label for text , imageview for image )
- continueDontEntyButton you maybe change the design like ( flexlayout( flexlayout for yellow line+ button or label for text )  
- callHelpButton ( headphone icon shuld be group and assign type image )
and add one more grup for imageview which use that image source . Right Arrow group and assign type image and add one more group for imageview. The text group and set type label or button. and all component select and group  set type flexlayout.

**Comment by fuatkestane** [comment]
Hi again,

I applied you said. For example: continueButton. Background color does not appear this item. I gave flexLayout set to item. Also, I can't see images. Because, the images folder is empty when exporting. 

Waiting for your help in urgent.

Smartface UI:
![screen shot 2019-01-18 at 10 18 20 am](https://user-images.githubusercontent.com/9901234/51371436-33d99b00-1b0b-11e9-917e-1b40f84fe2fb.png)
